### ISSUE TITLE
No wheels for Python 3.9+

### ISSUE THREAD

**Issue by kveretennicov** [brandtbucher/automap#9]
Repro:
> docker run --rm -it --entrypoint /bin/sh python:3.9.6-slim  # note the "slim"
> pip install automap

Expected: automap to install
Actual:
```
Collecting automap
  Downloading automap-0.4.8.tar.gz (10 kB)
Building wheels for collected packages: automap
  Building wheel for automap (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/local/bin/python -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-8ri60r3p/automap_9b098a1ffbca44c09a02efa7a7c9655c/setup.py'"'"'; __file__='"'"'/tmp/pip-install-8ri60r3p/automap_9b098a1ffbca44c09a02efa7a7c9655c/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-1nr45q6v
       cwd: /tmp/pip-install-8ri60r3p/automap_9b098a1ffbca44c09a02efa7a7c9655c/
  Complete output (8 lines):
  running bdist_wheel
  running build
  running build_ext
  building 'automap' extension
  creating build
  creating build/temp.linux-x86_64-3.9
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/local/include/python3.9 -c automap.c -o build/temp.linux-x86_64-3.9/automap.o
  error: command 'gcc' failed: No such file or directory
  ----------------------------------------
  ERROR: Failed building wheel for automap
```

This works with a non-slim image, but it is significantly larger than slim, which is a factor in cloud deployment. Seeing that automap wheels for 3.8 are published, I think it makes sense to extend that to 3.9 and other supported Python versions.

**Comment by kveretennicov** [comment]
@brandtbucher, I see that `ci.yml` has provisions for Python 3.9, yet 3.9 wheels are not in https://pypi.org/project/automap/0.4.8/#files. Is it just a matter of cutting new release?

**Comment by brandtbucher** [comment]
Yeah, thanks for the heads-up. I'll try to cut a new release sometime this week.

(There's quite a bit that's changed in the code since the last release, so I want to make sure that we don't have any performance/behavior regressions.)

Worst case I can just manually publish 3.9 and 3.10 wheels built from the last release. Do you personally need anything other than `
automap-0.4.8-cp39-cp39-manylinux1_x86_64.whl` on PyPI?

**Comment by kveretennicov** [comment]
Thanks! `automap-0.4.8-cp39-cp39-manylinux1_x86_64.whl` is what I personally need, though publishing "cp310" as well is probably a good idea.

FWIW, I am consuming `automap` via `static-frame`.

**Comment by brandtbucher** [comment]
Thanks for your patience! Automap 0.5.1 was just released, with 3.9 (and 3.10) wheels: https://pypi.org/project/automap/.
